### DISCUSSION TITLE
keyboard delay on oculus quest 1

### DISCUSSION THREAD

**Discussion by LOLZSICKA** [ppy/osu#21552] [Q&A]
### Type

Crash to desktop

### Bug description

slow inputs when doing chords in mania, definitely not the keyboard.

to reproduce: have a oculus quest 1 or 2, enable developer mode via a slightly complicated process (requires phone!), get sidequest on pc, install the osu!lazer apk for 2022.1123.0-lazer, play a 3star mania chart that contains chords (rungeon hard will work) using a wired keyboard of your choice (you might need a converter depending on what the keyboard uses for the ends). if done right, it should also have laggy inputs if you try pressing multiple buttons at once in quick sucession

### Version

osu! 2022.1123.0-lazer

### Logs

ill find the logs later, gonna play osu (im lazy)

**Comment by LOLZSICKA** [comment]
oof i forgot to set the report type

**Comment by smoogipoo** [comment]
This is a very esoteric setup and likely not going to be a target for performance for quite a while.
